lifelines/utils/sklearnAdapter.py

```diff
@@ -0,0 +1,84 @@
+import inspect
```

Comment:
my file name convention has been lower-case underscores. Can you duplicate that here? So `sklearn_adapter.py`

Reply:
feel free to do all the fixes you feel needed.